configuration/appframework/scripts/isPatientEnrolled.js

```diff
@@ -0,0 +1,6 @@
+function isPatientEnrolled(programs, programUuid) {
```

Comment:
Based on your logic here, I'd name this something like `isEverEnrolled`.  If this is intended to indicate if there is a _current_ / _active_ enrollment, then that is not being checked here.  `isPatientEnrolled` implies that they are currently enrolled.